02Lesson/01Homework/index.html

```diff
@@ -1,8 +1,72 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title></title>
-</head>
-<body>
-</body>
+  <head>
+    <title>My Bio</title>
+    <meta charset="utf-8">
+  </head>
+  <body>
+   <section>
+    <h1>
+      Travis Schlicke
+    </h1>
+    <p>
+      My address goes here
+    </p>
+  </section>
+    <h3>
+      Bio
+    </h3>
+    <p>
+      I am a man that believes to never stop learning. From learning how to give masssages to learning how to cook.
+      I have many things I have learned. My latest adventure is taking this computer class to learn how to code.
+      I am a man that is classified as a nerd and I love that I am. I am huge into video games and sports. Sports
+      is not a typical thing nerds enjoy but for me I love them. My favorites are Soccer and American Football. Go Cowboys.
+    </p>
+  <hr>
+  <div style="color:#0000FF">
+    <h3>
+      Hobbies
+    </h3>
+      <ol type="1">
+        <li>Vido Games especially Heroes of the Storm</li>
+        <li>Sports, mainly Soccer and American Football, both playing and watching</li>
+        <li>Playing Board Games</li>
+        <li>Anything with water, water parks, lakes, ocean etc..</li>
+      </ol>
+    </div>
+
+      <h3>
+        Things I don't like
+      </h3>
+      <ul>
+        <li> Slow drivers </li>
+        <li> Sweet Potatoes </li>
+        <li> People that lie </li>
+        <li> Playing Support in Herores of the Storm </li>
+
+      </ul>
+
+      <h3>
+        Shows I love
+      </h3>
+
+      <img class="irc_mi ihFMqKYz_r74-pQOPx8XEepE" style="margin-top: 37px;" onload="google.aft&amp;&amp;google.aft(this)" src="http://www.morphsuits.com/blog/wp-content/uploads/2015/07/game-of-thrones.jpg" width="600" height="400" alt="Image result for picture of game of thrones">
+      <p> GAME OF THRONES
+      </p>
```

Comment:
Indentation: I would always put block elements on their own line and indent the content: 
```html
<p>
  GAME OF THRONES
</p>
```
That prevents you from forgetting closing tags when the code gets more complex.